.github/workflows/main.yml

```
@@ -41,11 +67,16 @@ jobs:
 
       - name: Prepare data
         run: |
-          # pytest tests/test_specific_file.py::TestClass::test_function
           source activate TEST
           pytest mhkit/tests/river/test_io.py
           pytest mhkit/tests/tidal/test_io.py
           pytest mhkit/tests/wave/io/test_cdip.py
+
+      - name: Prepare Hindcast data
+        if: (github.event.pull_request.base.ref == 'master' || needs.check-changes.outputs.wave_io_hindcast_changed == 'true')
```

We should use the should-run-hindcast variable so that the test logic is not duplicated

```suggestion
        if: (needs.check-changes.outputs.should-run-hindcast == 'true')

---

Good call! Fixed in [b81f61f](https://github.com/MHKiT-Software/MHKiT-Python/pull/270/commits/b81f61f786960453af2449a016876285e45a2426)